vestido/vestido-storefront/modules/products/product-view.tsx

```diff
@@ -198,10 +227,27 @@ const ProductView: React.FC<ProductViewProps> = ({ itemId }) => {
               </>
             ) : (
               <div className="grid grid-cols-3 w-full">
-                <div className="flex flex-col  gap-1 items-center ">
-                  <LuScaling size={24} />
-                  <div className="text-xs">Size Guide</div>
-                </div>
+                <Dialog>
+                  <DialogTrigger asChild>
+                    <div className="flex flex-col  gap-1 items-center ">
+                      <LuScaling size={24} />
+                      <div className="text-xs">Size Guide</div>
+                    </div>
+                  </DialogTrigger>
+                  <DialogContent>
```

Comment:
Duplicate Dialog Components? Please make it one. Remove the Javascript based MediaQuery conditional.